apps/backoffice-v2/src/lib/zod/utils/checkers/index.ts

```diff
@@ -1,11 +1,11 @@
 import { isType } from '@ballerine/common';
 import { z } from 'zod';
 
+export const BooleanishSchema = z.record(
```

Comment:
Maybe change "Booleanish" to "BooleanString"?

Reply:
Booleanish is a recognized concept. Let me know what you think considering that. (from the React types)
![image](https://github.com/ballerine-io/ballerine/assets/61207713/cfbb1a0b-b05e-43ff-b89f-f4cc2a9b4293)